Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8-slim
+WORKDIR /mote
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+RUN sed -i 's|json_cache_location = "/var/cache/httpd/mote/cache.json"|json_cache_location = "./cache.json"|' ./mote/config.py
```

Comment:
Once, This [PR](https://github.com/fedora-infra/mote/pull/149`) is merged, we wouldn't have to modify it.

Reply:
Does it have a green flag yet?

If so, I can remove the changes made in the L#5 of Dockerfile in this PR to make way for #149.